test/JDBC/expected/BABEL-5242-union-tsql-basetype.out

```diff
@@ -0,0 +1,458 @@
+--
+-- Tests on the result data type should retain as tsql base type after UNION
+--
+CREATE TABLE t1 (
+  a INT,
+  TransactionAmount money NULL
+);
+GO
+
+SELECT c.[name] AS [sys_types_user_type_id],
+       TYPE_NAME(a.user_type_id) AS type_name_function_user_type_id
+FROM sys.columns a
+    INNER JOIN sys.objects b
+        ON a.[object_id] = b.[object_id]
+    LEFT JOIN sys.types c
+        ON a.user_type_id = c.user_type_id
+WHERE b.[name] = 't1'
+    AND a.[name] = 'TransactionAmount';
+GO
+~~START~~
+varchar#!#nvarchar
+money#!#money
+~~END~~
+
+
+CREATE VIEW TestView AS
+    select
+        traamo = a
+    from t1
+union all
+    select
+        traamo = TransactionAmount
+    from t1;
+GO
+
```

Comment:
we need to have some better coverage where union involves more than 2 block and has some variety of base/domain type combination